.buildkite/pipeline.yml

```diff
@@ -101,6 +101,12 @@ steps:
         agents:
           queue: hosted
         plugins:
+          - aws-assume-role-with-web-identity#v1.0.0:
+              role-arn: arn:aws:iam::445615400570:role/pipeline-buildkite-docs
+          - ecr#v2.9.0:
+              login: true
+              account-ids: public.ecr.aws
+              region: us-east-1
```

Comment:
do we need to specify the region? Or will the plugin just always use `us-east-1` for the public gallery?

![2024-06-21_15-01](https://github.com/buildkite/docs/assets/8132/2e7a310f-9688-4d48-bcac-1caae352be9f)